CMakeLists.txt

```diff
@@ -1,10 +1,275 @@
 # Native code side of Thrive
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.13)
+
+set(CMAKE_CONFIGURATION_TYPES "Debug;Release;RelWithDebInfo;Distribution" CACHE STRING "" FORCE)
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
```

Comment:
This should be just removed as this is default behaviour and I think it is unnecessarily complication to be at the top of the file.

Reply:
done :]